Script/AtomicEditor/ui/frames/inspector/InspectorFrame.ts

```diff
@@ -213,6 +214,22 @@ class InspectorFrame extends ScriptWidget {
             prefabInspector.inspect(asset);
         }
 
+        if (asset.importerTypeName == "TextureImporter") {
```

Comment:
Yeah, we'll definitely want to give this the ol' abstraction 1, 2.  This is great for now...